File: LC_CPP/9_Palindrome Number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x < 0)
            return false;
        int dst = 0, var = 0;;
        int src = x;
        while(src > 0){
            var = src % 10;
            dst = dst * 10 + var;
            src /= 10;
        }
        if(dst != x)
            return false;
        return true;
    }
};

File: LC_Python/9_Palindrome Number.py
class Solution:
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return False
        str_num = str(x)
        if str_num[::-1] == str_num:
            return True
        else:
            return False
        